### ISSUE TITLE
Option for tracker window to grow width to the left

### ISSUE THREAD

**Issue by SenorSmartyPants** [AeroScripts/QuestieDev#388]
I like to place my tracker window on the right side of my screen. If I get a really long quest it flows off the right edge of my screen. I'd rather have the option to keep the right side of the tracker frame fixed and have the right grow and shrink.

**Comment by SalviaSage** [comment]
I agree with the original poster.
Please add a switch feature so the grow can be either towards the right (like default) or left.

**Comment by Dyaxler** [comment]
I never thought of that! Well, I did when I wrote the code for the Tracker Tool tip. It switches sides if the player puts the tracker on the right. 

Hmmm.... the problem with making the back ground shift to the left when the whole tracker is on the right is that human language is written exclusively from left to right. So the anchor point for the tracker is on the [BOTTOM_LEFT]/[BOTTOM_LEFT] and the buttons that fill it are all justified [LEFT]. The whole tracker back ground would still shift left and right as you add or remove quests. I'll try and justify the entire trackers button text [RIGHT] and changing the anchor point to [BOTTOM_RIGHT] and see how it looks. The anchor point is important otherwise you wouldn't be able to move the tracker around on your screen. If I can't change the anchor point to [BOTTOM_RIGHT]/[TOP_RIGHT]

NOTE: The current trackers anchor point (top or bottom) depends on the /questie listdirection command.

**Comment by SenorSmartyPants** [comment]
Keep all the text justified settings the same, just reposition the whole frame.

**Comment by Dyaxler** [comment]
I let this one slip under the radar with all the other features we have cooking. I've given this some more thought and now I think I might have a better way to handle the width of the tracker. I can set a hard width and make the text wrap forcing the tracker to grow upwards or downwards exclusively. That way we won't have to worry about justifying anything and it'll look cleaner. This would get rid of a lot of resizing code and some overhead since I won't be dumping as many tables into garbage collection.

**Comment by Dyaxler** [comment]
OK, THAT didn't work so well... :P

But I did come up with something that I think everyone will be happy with. I wrote a quick little block of code that will detect your screen width. When you drag the tracker to the right side of your screen it will change it's "setpoint" from bottom left to bottom right. Once you have your tracker where you want, do a quick ReloadUI() to "lock" it into position. This setting will stay locked and persistent across gaming sessions until you move it again. Otherwise, your tracker could potentially "crawl" across your screen if you add or remove quests from the tracker. 

By changing the setpoint to bottom right when the tracker is on the right side of your screen, if you obtain a quest that causes the tracker to resize wider or narrower, it'll resize then move your tracker to its new setpoint automatically so the bottom right corner (or top right corner depending on which direction you're listing your quests) will always stay where you put/locked it.
